Klaijan/improve is_possible_narrative_text and is_possible_title (#1309)
Unstructured-IO/unstructured: CHANGELOG.md

### Enhancements

* Updated documentation: Added back support doc types for partitioning, more Python codes in the API page,  RAG definition, and use case.

> * Updated `is_possible_title` detection rule

Reviewer: maybe worth writing a bit more on what changes are introduced?

Author: On it!

### Enhancements

* Updated documentation: Added back support doc types for partitioning, more Python codes in the API page,  RAG definition, and use case.

> * Updated `is_possible_title` detection rule by adding condition that title would not ends with period, but can ends with 3 dots

Reviewer 2: ```suggestion
* Updated `is_possible_title` detection rule by adding condition that title does not end with period, but can end with 3 dots
```
